### ISSUE TITLE
--statistics stopped working in 3.7.0

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#675]
In GitLab by @oskarpersson on Jan 30, 2019, 14:02

*Please describe how you installed Flake8*

```
$ pip install flake8
```

*Please provide the exact, unmodified output of `flake8 --bug-report`*

```
{
  "dependencies": [
    {
      "dependency": "entrypoints",
      "version": "0.3"
    }
  ],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.6.7",
    "system": "Darwin"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.5.0"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "2.1.0"
    }
  ],
  "version": "3.7.2"
}
```

*Please describe the problem or feature*

`flake8 --statistics` no longer prints any statistics at the end of the output. It stopped working in 3.7.0.

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Jan 30, 2019, 14:10

@asottile I suspect this is related to `--per-file-ignores`

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 30, 2019, 14:18

I do too, I think I've found the fix for it, writing a test :D

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 30, 2019, 14:40

mentioned in merge request !295

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 30, 2019, 14:41

!295 should have a fix for this

here's my mini validation (that I replicated in an integration test):

```
$ ~/opt/venv/bin/flake8 ../t.py  --statistics
../t.py:1:1: F401 'os' imported but unused
../t.py:2:1: F401 'sys' imported but unused
$ ./.tox/py36/bin/flake8 ../t.py --statistics
../t.py:1:1: F401 'os' imported but unused
../t.py:2:1: F401 'sys' imported but unused
2     F401 'os' imported but unused
```

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 30, 2019, 15:05

closed via merge request !295

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 30, 2019, 15:05

mentioned in commit cbffa49a0ba6a6b31cd61fd8009da2c9bb8386fd

**Comment by asottile** [comment]
In GitLab by @asottile on Jan 30, 2019, 19:58

This has been fixed in 3.7.3 -- thanks again for the issue :tada:
